src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/ProcessOutputLineListener.java

```diff
@@ -17,9 +17,13 @@
 package com.google.cloud.tools.app.impl.cloudsdk.internal.process;
 
 /**
- * Process listener that can be used with {@link DefaultProcessRunner} to read the process std or
- * error output line by line.
+ * Process listener that can be used with {@link DefaultProcessRunner} to read the merged stdout
```

Comment:
They're not merged. You have to register a separate listener for stdout and stderr.